src/common/src/metrics/guarded_metrics.rs

```diff
@@ -208,6 +208,14 @@ impl<T: MetricVecBuilder, const N: usize> LabelGuardedMetricVec<T, N> {
         }
     }
 
+    pub fn reset(&self) {
```

Comment:
Where is it used?

Reply:
https://github.com/risingwavelabs/risingwave/blob/main/src/meta/src/rpc/metrics.rs#L768